.travis.yml

```diff
@@ -149,6 +149,11 @@ stages:
 branches:
   only: [master]
 
+# MANUALLY ADDED. Do not remove.
+after_success:
```

Comment:
Will this run after every green travis run, even just for a pending PR that we have not actually approved yet?

Reply:
No, only after merged. The naming is meh, not my choice.